Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -331,11 +331,16 @@ void UpdateTabIcons()
 					continue;
 
 				TabLayout.Tab tab = tabs.GetTabAt(i);
-				tab.SetIcon(ResourceManager.IdFromTitle(icon, ResourceManager.DrawableClass));
+			    SetTabIcon(tab, icon);
 			}
 		}
 
-		void UpdateBarBackgroundColor()
+	    protected virtual void SetTabIcon(TabLayout.Tab tab, FileImageSource icon)
+	    {
+            tab.SetIcon(ResourceManager.IdFromTitle(icon, ResourceManager.DrawableClass));
+        }
+
+        void UpdateBarBackgroundColor()
```

Comment:
you are right, don't know why git changed this. What is the correct setting for git: update cr/lf to cr only ? replace space with tabs ?